src/components/ProductTable.jsx

```diff
@@ -0,0 +1,26 @@
+import ProductRow from "./ProductRow"
+
+
+
+function ProductTable(props) {
+    return (
+        <div style={{marginLeft: "795px", paddingTop: "20px"}}>
+
+            <table>
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Price</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {props.products.map((product, index) => (
+                        <ProductRow key={index} product={product}></ProductRow>
```

Comment:
Remember when using the key, its better to use the id in this case of the product.  also yo ucan close the tag in the same line

```suggestion
                        <ProductRow key={product.id} product={product}/>
```